test/case/convention/openshift/golang-osd-operator/05-build-olm-catalog

```diff
@@ -0,0 +1,210 @@
+#!/bin/bash
+
+# shellcheck disable=SC2207,SC2155
+
+set -euo pipefail
+
+export OLM_BUNDLE_IMAGE=quay.io/rporresm/deployment-validation-operator-bundle
+export OLM_CATALOG_IMAGE=quay.io/rporresm/deployment-validation-operator-catalog
+export CONTAINER_ENGINE=/usr/local/bin/docker
+export CONTAINER_ENGINE_CONFIG_DIR=.docker
+export CURRENT_COMMIT=860gta5q
+export COMMIT_NUMBER=5
+export OPERATOR_VERSION=0.1.5-860gta5q
+export OPERATOR_NAME=deployment-validation-operator
+export OPERATOR_IMAGE=quay.io/rporresm/deployment-validation-operator
+export OPERATOR_IMAGE_TAG=not-important-at-the-moment
+export OLM_BUNDLE_VERSIONS_REPO=gitlab.cee.redhat.com/service/saas-deployment-validation-operator-bundle.git
```

Comment:
Shouldn't it be "gitlab.cee.redhat.com/service/saas-operator-versions.git" also here ?

Reply:
we don't really care... it is not used in the unit tests (and should not be used 😄 ). I will change it to be `not-important-at-the-moment` to further clarify it

Reply:
fixed in cc6b3f6